### DISCUSSION TITLE
等级标签设置了无用

### DISCUSSION THREAD

**Discussion by gumpBOBO** [walinejs/waline#1729] [Q&A]
ming大大好。
我按文档设置了等级标签但是好像没有用，具体如下：

前端版本：  v2.6.1，本地我改成最新的v2.14.5也没用 
部署： vercel
数据库： data
服务端版本：@waline/admin  v0.23.1   （不知道是不是最新，进后台没提示我更新）
环境变量： LEVELS  0,5,20,50,100,200
yml配置
waline: 
 locale: {
    level0: '炼体',
    level1: '炼气',
    level2: '筑基',
    level3: '金丹',
    level4: '元婴',
    level5: '化神',
  }

博客页面能显示第一级的 炼体。 但是后面的就一直没出现过。

blog,.ganxb2.com

**Comment by gumpBOBO** [comment]
大概知道问题了 leancloud 中有个表 cache_group_count_user_id_mail
为啥deta中没有啊@lizheming

**Reply by lizheming** [reply]
deta 数据库功能受限，不支持这些功能
